python/fedml/api/modules/storage.py

```diff
@@ -10,20 +10,32 @@
 from fedml.api.fedml_response import FedMLResponse, ResponseCode
 
 
-# Todo (alaydshah): Add file size
 class StorageMetadata(object):
     def __init__(self, data: dict):
         self.dataName = data.get("datasetName", None)
         self.description = data.get("description", None)
         self.tags = data.get("description", None)
         self.createdAt = data.get("createTime", None)
         self.updatedAt = data.get("updateTime", None)
-
-
-# Todo (alaydshah): Add file size while creating objects. Store service name in metadata
+        size_in_bytes = data.get("fileSize", None)
+        if(size_in_bytes):
+            size = int(size_in_bytes)
+            size_in_gb = size / (1024 * 1024 * 1024)
+            size_in_mb = size / (1024 * 1024)
+            size_in_kb = size / 1024
+            if(size_in_gb >= 1):
+                self.size = f"{size_in_gb:.2f} GB"
+            elif(size_in_mb >= 1):
+                self.size = f"{size_in_mb:.2f} MB"
+            elif(size_in_kb >= 1):
+                self.size = f"{size_in_kb:.2f} KB"
+            else:
+                self.size = f"{size} B"
+
```

Comment:
How about moving this logic out in a private function?

Reply:
Done!